plugins/cue_points/quiz/lib/model/kQuizManager.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * @package plugins.quiz
+ */
+class kQuizManager implements kObjectAddedEventConsumer, kObjectChangedEventConsumer
+{
+
+	/* (non-PHPdoc)
+	 * @see kObjectAddedEventConsumer::shouldConsumeAddedEvent()
+	 */
+	public function shouldConsumeAddedEvent(BaseObject $object)
+	{
+		if($object instanceof AnswerCuePoint)
+			return true;
+
+		return false;
+	}
+
+	/* (non-PHPdoc)
+	 * @see kObjectChangedEventConsumer::shouldConsumeChangedEvent()
+	 */
+	public function shouldConsumeChangedEvent(BaseObject $object, array $modifiedColumns)
+	{
+		if( $object instanceof AnswerCuePoint
+			&& in_array(CuePointPeer::CUSTOM_DATA, $modifiedColumns)
+			&& $object->isCustomDataModified(AnswerCuePoint::CUSTOM_DATA_ANSWER_KEY) )
```

Comment:
I think that checking it in event consumer is wrong because the object may change by a previous consumer on the same event. You can check it in postSave on the object itself before raising the event.